.github/CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# --- GitHub configuration ---
+.github/CODEOWNERS                    @Health-Education-England/trainee-backend
+.github/workflows/*                   @Health-Education-England/trainee-devops
```

Comment:
You need to be careful of how the matchers work, this will match only the immediate children and not anything nested.

e.g. `.github/workflows/x.yml` would match but `.github/workflows/sub/y.xml` would not.